packaging/hassio/config.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "evcc",
+  "version": "0.100",
```

Comment:
Das matcht Deine regex nicht. Besser

    0.0.0

Reply:
Gut ist jetzt schwer dir zu wiedersprechen. 

Was ich getestet habe:

Tag:
v0.100.1 -> geht
v0.99.1 -> geht
v1.0.23 -> geht
v0.0.1 -> geht
v1 -> geht
v0.1 -> geht
1 -> geht
1.22.1 -> geht
22.33 -> geht

Das Tool schreibt den Tag value ohne "v" (wenn vorhanden) in die "" hinter `"version": `